background.js

```diff
@@ -1,27 +1,28 @@
-chrome.runtime.onInstalled.addListener(() => {
-  chrome.storage.local.set({ seenArticles: [] });
+globalThis.browser ??= chrome;
```

Comment:
A minimal polyfill that enables the use of the common cross-browser global namespace for WebExtensions.